packages/truffle-db/src/workspace/test/utils.ts

```diff
@@ -16,22 +18,38 @@ export const fixturesDirectory = path.join(
   "fixtures"
 );
 
+const tempDir = tmp.dirSync({ unsafeCleanup: true });
+tmp.setGracefulCleanup();
+
 export class WorkspaceClient {
+  // two workspaces are generated here, one is empty for adding data, and one is queried using
+  // the data that should have been persisted in the .db directory when data was added
   private workspace: Workspace;
+  private persistedWorkspace: Workspace;
 
   constructor () {
-    this.workspace = new Workspace();
+    this.workspace = new Workspace(tempDir.name);
+  }
+
+  async destroy(resource) {
+    await this.workspace.dbApi.bytecodes.destroy();
```

Comment:
You have this hardcoded as `bytecodes` instead of `resource`

Reply:
Thanks! Fixed.